sql: TestParallelCreateTables failed under stress

SHA: https://github.com/cockroachdb/cockroach/commits/298907ef2a3d33cd87e255b663d76c8eb74a62df

Parameters:
```
COCKROACH_PROPOSER_EVALUATED_KV=false
TAGS=
GOFLAGS=
```

Stress build found a failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=144530&tab=buildLog

```
W170209 08:16:46.355017 14244 server/status/runtime.go:116  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I170209 08:16:46.355952 14244 server/config.go:482  1 storage engine initialized
I170209 08:16:46.356159 14244 server/node.go:442  [n?] store [n0,s0] not bootstrapped
I170209 08:16:46.357662 14244 server/node.go:371  [n?] **** cluster 6b903678-546b-4d5b-bce0-fa58a5b7d153 has been created
I170209 08:16:46.357684 14244 server/node.go:372  [n?] **** add additional nodes by specifying --join=127.0.0.1:34680
I170209 08:16:46.358118 14244 storage/store.go:1261  [n1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I170209 08:16:46.358148 14244 server/node.go:455  [n1] initialized store [n1,s1]: {Capacity:536870912 Available:536870912 RangeCount:1 LeaseCount:0}
I170209 08:16:46.358170 14244 server/node.go:340  [n1] node ID 1 initialized
I170209 08:16:46.358213 14244 gossip/gossip.go:293  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:34680" > attrs:<> locality:<> 
I170209 08:16:46.358319 14244 storage/stores.go:296  [n1] read 0 node addresses from persistent storage
I170209 08:16:46.358358 14244 server/node.go:587  [n1] connecting to gossip network to verify cluster ID...
I170209 08:16:46.359134 14244 server/node.go:611  [n1] node connected via gossip and verified as part of cluster "6b903678-546b-4d5b-bce0-fa58a5b7d153"
I170209 08:16:46.359163 14244 server/node.go:390  [n1] node=1: started with [[]=] engine(s) and attributes []
I170209 08:16:46.359192 14244 sql/executor.go:332  [n1] creating distSQLPlanner with address {tcp 127.0.0.1:34680}
I170209 08:16:46.360973 14244 server/server.go:632  [n1] starting https server at 127.0.0.1:60316
I170209 08:16:46.360993 14244 server/server.go:633  [n1] starting grpc/postgres server at 127.0.0.1:34680
I170209 08:16:46.361006 14244 server/server.go:634  [n1] advertising CockroachDB node at 127.0.0.1:34680
I170209 08:16:46.361585 14339 storage/split_queue.go:98  [split,n1,s1,r1/1:/M{in-ax},@c421a74a80] splitting at key /Table/0/0
I170209 08:16:46.362963 14339 storage/replica_command.go:2397  [split,n1,s1,r1/1:/M{in-ax},@c421a74a80] initiating a split of this range at key /Table/0 [r2]
I170209 08:16:46.366741 14349 sql/event_log.go:95  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:34680} Attrs: Locality:} ClusterID:6b903678-546b-4d5b-bce0-fa58a5b7d153 StartedAt:1486628206359149785}
I170209 08:16:46.368918 14244 sql/event_log.go:95  [n1] Event: "alter_table", target: 12, info: {TableName:eventlog Statement:ALTER TABLE system.eventlog ALTER COLUMN uniqueID SET DEFAULT uuid_v4() User:node MutationID:0 CascadeDroppedViews:[]}
I170209 08:16:46.374138 14339 storage/split_queue.go:98  [split,n1,s1,r2/1:/{Table/0-Max},@c420838e00] splitting at key /Table/11/0
I170209 08:16:46.374179 14339 storage/replica_command.go:2397  [split,n1,s1,r2/1:/{Table/0-Max},@c420838e00] initiating a split of this range at key /Table/11 [r3]
I170209 08:16:46.381356 14244 server/server.go:689  [n1] done ensuring all necessary migrations have run
I170209 08:16:46.381373 14244 server/server.go:691  [n1] serving sql connections
I170209 08:16:46.388871 14339 storage/split_queue.go:98  [split,n1,s1,r3/1:/{Table/11-Max},@c421a44e00] splitting at key /Table/12/0
I170209 08:16:46.388903 14339 storage/replica_command.go:2397  [split,n1,s1,r3/1:/{Table/11-Max},@c421a44e00] initiating a split of this range at key /Table/12 [r4]
I170209 08:16:46.404533 14339 storage/split_queue.go:98  [split,n1,s1,r4/1:/{Table/12-Max},@c421af4700] splitting at key /Table/13/0
I170209 08:16:46.404568 14339 storage/replica_command.go:2397  [split,n1,s1,r4/1:/{Table/12-Max},@c421af4700] initiating a split of this range at key /Table/13 [r5]
I170209 08:16:46.413676 14339 storage/split_queue.go:98  [split,n1,s1,r5/1:/{Table/13-Max},@c421a45500] splitting at key /Table/14/0
I170209 08:16:46.413709 14339 storage/replica_command.go:2397  [split,n1,s1,r5/1:/{Table/13-Max},@c421a45500] initiating a split of this range at key /Table/14 [r6]
W170209 08:16:46.452185 14244 gossip/gossip.go:1178  [n?] no incoming or outgoing connections
W170209 08:16:46.458708 14244 server/status/runtime.go:116  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I170209 08:16:46.471345 14244 server/config.go:482  1 storage engine initialized
I170209 08:16:46.471545 14244 server/node.go:442  [n?] store [n0,s0] not bootstrapped
I170209 08:16:46.471563 14244 storage/stores.go:296  [n?] read 0 node addresses from persistent storage
I170209 08:16:46.471581 14244 server/node.go:587  [n?] connecting to gossip network to verify cluster ID...
I170209 08:16:46.473401 14545 gossip/client.go:131  [n?] started gossip client to 127.0.0.1:34680
I170209 08:16:46.474547 14579 gossip/server.go:214  [n1] received initial cluster-verification connection from {tcp 127.0.0.1:42865}
I170209 08:16:46.474933 14244 server/node.go:611  [n?] node connected via gossip and verified as part of cluster "6b903678-546b-4d5b-bce0-fa58a5b7d153"
I170209 08:16:46.475834 14569 storage/stores.go:312  [n?] wrote 1 node addresses to persistent storage
I170209 08:16:46.476773 14244 kv/dist_sender.go:367  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I170209 08:16:46.477251 14244 server/node.go:333  [n?] new node allocated ID 2
I170209 08:16:46.477295 14244 gossip/gossip.go:293  [n2] NodeDescriptor set to node_id:2 address:<network_field:"tcp" address_field:"127.0.0.1:42865" > attrs:<> locality:<> 
I170209 08:16:46.477337 14244 server/node.go:390  [n2] node=2: started with [[]=] engine(s) and attributes []
I170209 08:16:46.477360 14244 sql/executor.go:332  [n2] creating distSQLPlanner with address {tcp 127.0.0.1:42865}
I170209 08:16:46.479379 14244 server/server.go:632  [n2] starting https server at 127.0.0.1:59021
I170209 08:16:46.479397 14244 server/server.go:633  [n2] starting grpc/postgres server at 127.0.0.1:42865
I170209 08:16:46.479409 14244 server/server.go:634  [n2] advertising CockroachDB node at 127.0.0.1:42865
I170209 08:16:46.493412 14415 storage/stores.go:312  [n1] wrote 1 node addresses to persistent storage
I170209 08:16:46.493896 14244 server/server.go:689  [n2] done ensuring all necessary migrations have run
I170209 08:16:46.493915 14244 server/server.go:691  [n2] serving sql connections
I170209 08:16:46.497212 14595 server/node.go:568  [n2] bootstrapped store [n2,s2]
I170209 08:16:46.498955 14419 storage/replica_raftstorage.go:414  [replicate,n1,s1,r5/1:/Table/1{3-4},@c421a45500] generated preemptive snapshot da848636 at index 16
W170209 08:16:46.499512 14244 gossip/gossip.go:1178  [n?] no incoming or outgoing connections
W170209 08:16:46.500127 14244 server/status/runtime.go:116  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I170209 08:16:46.501033 14244 server/config.go:482  1 storage engine initialized
I170209 08:16:46.501255 14244 server/node.go:442  [n?] store [n0,s0] not bootstrapped
I170209 08:16:46.501268 14244 storage/stores.go:296  [n?] read 0 node addresses from persistent storage
I170209 08:16:46.501282 14244 server/node.go:587  [n?] connecting to gossip network to verify cluster ID...
I170209 08:16:46.503307 14598 sql/event_log.go:95  [n2] Event: "node_join", target: 2, info: {Descriptor:{NodeID:2 Address:{NetworkField:tcp AddressField:127.0.0.1:42865} Attrs: Locality:} ClusterID:6b903678-546b-4d5b-bce0-fa58a5b7d153 StartedAt:1486628206477326275}
I170209 08:16:46.509215 14419 storage/store.go:3278  [replicate,n1,s1,r5/1:/Table/1{3-4},@c421a45500] streamed snapshot: kv pairs: 40, log entries: 6, 1ms
I170209 08:16:46.509460 14558 storage/replica_raftstorage.go:596  [n2,s2,r5/?:{-},@c4216b0a80] applying preemptive snapshot at index 16 (id=da848636, encoded size=5616, 1 rocksdb batches, 6 log entries)
I170209 08:16:46.509654 14558 storage/replica_raftstorage.go:604  [n2,s2,r5/?:/Table/1{3-4},@c4216b0a80] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170209 08:16:46.510281 14738 gossip/client.go:131  [n?] started gossip client to 127.0.0.1:34680
I170209 08:16:46.510593 14771 gossip/server.go:214  [n1] received initial cluster-verification connection from {tcp 127.0.0.1:47786}
I170209 08:16:46.510696 14244 server/node.go:611  [n?] node connected via gossip and verified as part of cluster "6b903678-546b-4d5b-bce0-fa58a5b7d153"
I170209 08:16:46.510891 14759 storage/stores.go:312  [n?] wrote 1 node addresses to persistent storage
I170209 08:16:46.510923 14759 storage/stores.go:312  [n?] wrote 2 node addresses to persistent storage
I170209 08:16:46.511284 14419 storage/replica_command.go:3253  [replicate,n1,s1,r5/1:/Table/1{3-4},@c421a45500] change replicas (remove {2 2 2}): read existing descriptor range_id:5 start_key:"\225" end_key:"\226" replicas:<node_id:1 store_id:1 replica_id:1 > next_replica_id:2 
I170209 08:16:46.514079 14244 kv/dist_sender.go:367  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I170209 08:16:46.514137 14786 storage/replica.go:2476  [n1,s1,r5/1:/Table/1{3-4},@c421a45500] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2}]
I170209 08:16:46.514636 14244 server/node.go:333  [n?] new node allocated ID 3
I170209 08:16:46.514678 14244 gossip/gossip.go:293  [n3] NodeDescriptor set to node_id:3 address:<network_field:"tcp" address_field:"127.0.0.1:47786" > attrs:<> locality:<> 
I170209 08:16:46.514711 14419 storage/replica_raftstorage.go:414  [replicate,n1,s1,r3/1:/Table/1{1-2},@c421a44e00] generated preemptive snapshot 6eeb3705 at index 22
I170209 08:16:46.514723 14244 server/node.go:390  [n3] node=3: started with [[]=] engine(s) and attributes []
I170209 08:16:46.514739 14244 sql/executor.go:332  [n3] creating distSQLPlanner with address {tcp 127.0.0.1:47786}
I170209 08:16:46.515230 14419 storage/store.go:3278  [replicate,n1,s1,r3/1:/Table/1{1-2},@c421a44e00] streamed snapshot: kv pairs: 10, log entries: 12, 0ms
I170209 08:16:46.515516 14781 storage/raft_transport.go:437  [n2] raft transport stream to node 1 established
I170209 08:16:46.515751 14790 storage/replica_raftstorage.go:596  [n2,s2,r3/?:{-},@c42266e000] applying preemptive snapshot at index 22 (id=6eeb3705, encoded size=6038, 1 rocksdb batches, 12 log entries)
I170209 08:16:46.515886 14708 storage/stores.go:312  [n1] wrote 2 node addresses to persistent storage
I170209 08:16:46.515946 14790 storage/replica_raftstorage.go:604  [n2,s2,r3/?:/Table/1{1-2},@c42266e000] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170209 08:16:46.516076 14709 storage/stores.go:312  [n2] wrote 2 node addresses to persistent storage
I170209 08:16:46.516330 14419 storage/replica_command.go:3253  [replicate,n1,s1,r3/1:/Table/1{1-2},@c421a44e00] change replicas (remove {2 2 2}): read existing descriptor range_id:3 start_key:"\223" end_key:"\224" replicas:<node_id:1 store_id:1 replica_id:1 > next_replica_id:2 
I170209 08:16:46.516690 14244 server/server.go:632  [n3] starting https server at 127.0.0.1:57920
I170209 08:16:46.516708 14244 server/server.go:633  [n3] starting grpc/postgres server at 127.0.0.1:47786
I170209 08:16:46.516716 14244 server/server.go:634  [n3] advertising CockroachDB node at 127.0.0.1:47786
I170209 08:16:46.518304 14244 server/server.go:689  [n3] done ensuring all necessary migrations have run
I170209 08:16:46.518323 14244 server/server.go:691  [n3] serving sql connections
I170209 08:16:46.532019 14683 server/node.go:568  [n3] bootstrapped store [n3,s3]
I170209 08:16:46.532422 14811 storage/replica.go:2476  [n1,s1,r3/1:/Table/1{1-2},@c421a44e00] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2}]
I170209 08:16:46.533496 14419 storage/replica_raftstorage.go:414  [replicate,n1,s1,r4/1:/Table/1{2-3},@c421af4700] generated preemptive snapshot a43c23ed at index 20
I170209 08:16:46.534143 14817 storage/replica_raftstorage.go:596  [n2,s2,r4/?:{-},@c4216b0e00] applying preemptive snapshot at index 20 (id=a43c23ed, encoded size=8479, 1 rocksdb batches, 10 log entries)
I170209 08:16:46.534345 14817 storage/replica_raftstorage.go:604  [n2,s2,r4/?:/Table/1{2-3},@c4216b0e00] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170209 08:16:46.534632 14686 sql/event_log.go:95  [n3] Event: "node_join", target: 3, info: {Descriptor:{NodeID:3 Address:{NetworkField:tcp AddressField:127.0.0.1:47786} Attrs: Locality:} ClusterID:6b903678-546b-4d5b-bce0-fa58a5b7d153 StartedAt:1486628206514708621}
I170209 08:16:46.535134 14419 storage/store.go:3278  [replicate,n1,s1,r4/1:/Table/1{2-3},@c421af4700] streamed snapshot: kv pairs: 36, log entries: 10, 0ms
I170209 08:16:46.537904 14419 storage/replica_command.go:3253  [replicate,n1,s1,r4/1:/Table/1{2-3},@c421af4700] change replicas (remove {2 2 2}): read existing descriptor range_id:4 start_key:"\224" end_key:"\225" replicas:<node_id:1 store_id:1 replica_id:1 > next_replica_id:2 
I170209 08:16:46.540752 14923 storage/replica.go:2476  [n1,s1,r4/1:/Table/1{2-3},@c421af4700] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2}]
I170209 08:16:46.541276 14419 storage/replica_raftstorage.go:414  [replicate,n1,s1,r2/1:/Table/{0-11},@c420838e00] generated preemptive snapshot 82e14f8b at index 24
I170209 08:16:46.552915 14419 storage/store.go:3278  [replicate,n1,s1,r2/1:/Table/{0-11},@c420838e00] streamed snapshot: kv pairs: 33, log entries: 14, 2ms
I170209 08:16:46.553187 14950 storage/replica_raftstorage.go:596  [n3,s3,r2/?:{-},@c42094ee00] applying preemptive snapshot at index 24 (id=82e14f8b, encoded size=12461, 1 rocksdb batches, 14 log entries)
I170209 08:16:46.553486 14950 storage/replica_raftstorage.go:604  [n3,s3,r2/?:/Table/{0-11},@c42094ee00] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170209 08:16:46.555021 14419 storage/replica_command.go:3253  [replicate,n1,s1,r2/1:/Table/{0-11},@c420838e00] change replicas (remove {3 3 2}): read existing descriptor range_id:2 start_key:"\210" end_key:"\223" replicas:<node_id:1 store_id:1 replica_id:1 > next_replica_id:2 
I170209 08:16:46.558573 14969 storage/replica.go:2476  [n1,s1,r2/1:/Table/{0-11},@c420838e00] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:3 StoreID:3 ReplicaID:2}]
I170209 08:16:46.559128 14419 storage/replica_raftstorage.go:414  [replicate,n1,s1,r1/1:/{Min-Table/0},@c421a74a80] generated preemptive snapshot 56ac0ade at index 67
I170209 08:16:46.563259 14419 storage/store.go:3278  [replicate,n1,s1,r1/1:/{Min-Table/0},@c421a74a80] streamed snapshot: kv pairs: 783, log entries: 57, 4ms
I170209 08:16:46.563503 14954 storage/replica_raftstorage.go:596  [n3,s3,r1/?:{-},@c42027ca80] applying preemptive snapshot at index 67 (id=56ac0ade, encoded size=255981, 1 rocksdb batches, 57 log entries)
I170209 08:16:46.563997 14978 storage/raft_transport.go:437  [n3] raft transport stream to node 1 established
I170209 08:16:46.586698 14954 storage/replica_raftstorage.go:604  [n3,s3,r1/?:/{Min-Table/0},@c42027ca80] applied preemptive snapshot in 23ms [clear=0ms batch=0ms entries=3ms commit=3ms]
I170209 08:16:46.587593 14419 storage/replica_command.go:3253  [replicate,n1,s1,r1/1:/{Min-Table/0},@c421a74a80] change replicas (remove {3 3 2}): read existing descriptor range_id:1 start_key:"" end_key:"\210" replicas:<node_id:1 store_id:1 replica_id:1 > next_replica_id:2 
I170209 08:16:46.591791 14897 storage/replica.go:2476  [n1,s1,r1/1:/{Min-Table/0},@c421a74a80] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:3 StoreID:3 ReplicaID:2}]
I170209 08:16:46.592820 14419 storage/replica_raftstorage.go:414  [replicate,n1,s1,r6/1:/{Table/14-Max},@c420839500] generated preemptive snapshot 0cdf3695 at index 11
I170209 08:16:46.594325 14938 storage/replica_raftstorage.go:596  [n3,s3,r6/?:{-},@c42219bc00] applying preemptive snapshot at index 11 (id=0cdf3695, encoded size=504, 1 rocksdb batches, 1 log entries)
I170209 08:16:46.594489 14938 storage/replica_raftstorage.go:604  [n3,s3,r6/?:/{Table/14-Max},@c42219bc00] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170209 08:16:46.594968 14419 storage/store.go:3278  [replicate,n1,s1,r6/1:/{Table/14-Max},@c420839500] streamed snapshot: kv pairs: 9, log entries: 1, 2ms
I170209 08:16:46.599280 14419 storage/replica_command.go:3253  [replicate,n1,s1,r6/1:/{Table/14-Max},@c420839500] change replicas (remove {3 3 2}): read existing descriptor range_id:6 start_key:"\226" end_key:"\377\377" replicas:<node_id:1 store_id:1 replica_id:1 > next_replica_id:2 
I170209 08:16:46.633671 15008 storage/replica.go:2476  [n1,s1,r6/1:/{Table/14-Max},@c420839500] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:3 StoreID:3 ReplicaID:2}]
I170209 08:16:46.634074 14419 storage/queue.go:693  [n1,replicate] purgatory is now empty
I170209 08:16:46.634302 14340 storage/replica_raftstorage.go:414  [replicate,n1,s1,r5/1:/Table/1{3-4},@c421a45500] generated preemptive snapshot 9d5eb423 at index 29
I170209 08:16:46.635797 15012 storage/replica_raftstorage.go:596  [n3,s3,r5/?:{-},@c421ae6e00] applying preemptive snapshot at index 29 (id=9d5eb423, encoded size=17194, 1 rocksdb batches, 19 log entries)
I170209 08:16:46.636066 15012 storage/replica_raftstorage.go:604  [n3,s3,r5/?:/Table/1{3-4},@c421ae6e00] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170209 08:16:46.636183 14340 storage/store.go:3278  [replicate,n1,s1,r5/1:/Table/1{3-4},@c421a45500] streamed snapshot: kv pairs: 76, log entries: 19, 1ms
I170209 08:16:46.636549 14340 storage/replica_command.go:3253  [replicate,n1,s1,r5/1:/Table/1{3-4},@c421a45500] change replicas (remove {3 3 3}): read existing descriptor range_id:5 start_key:"\225" end_key:"\226" replicas:<node_id:1 store_id:1 replica_id:1 > replicas:<node_id:2 store_id:2 replica_id:2 > next_replica_id:3 
I170209 08:16:46.640482 15029 storage/replica.go:2476  [n1,s1,r5/1:/Table/1{3-4},@c421a45500] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2} {NodeID:3 StoreID:3 ReplicaID:3}]
I170209 08:16:46.645015 14340 storage/replica_raftstorage.go:414  [replicate,n1,s1,r1/1:/{Min-Table/0},@c421a74a80] generated preemptive snapshot 8c42d8eb at index 78
I170209 08:16:46.651586 14340 storage/store.go:3278  [replicate,n1,s1,r1/1:/{Min-Table/0},@c421a74a80] streamed snapshot: kv pairs: 789, log entries: 68, 6ms
I170209 08:16:46.651923 15032 storage/replica_raftstorage.go:596  [n2,s2,r1/?:{-},@c421f29880] applying preemptive snapshot at index 78 (id=8c42d8eb, encoded size=259557, 1 rocksdb batches, 68 log entries)
I170209 08:16:46.672082 15032 storage/replica_raftstorage.go:604  [n2,s2,r1/?:/{Min-Table/0},@c421f29880] applied preemptive snapshot in 20ms [clear=0ms batch=0ms entries=19ms commit=1ms]
I170209 08:16:46.673649 14340 storage/replica_command.go:3253  [replicate,n1,s1,r1/1:/{Min-Table/0},@c421a74a80] change replicas (remove {2 2 3}): read existing descriptor range_id:1 start_key:"" end_key:"\210" replicas:<node_id:1 store_id:1 replica_id:1 > replicas:<node_id:3 store_id:3 replica_id:2 > next_replica_id:3 
I170209 08:16:46.677903 14985 storage/replica.go:2476  [n1,s1,r1/1:/{Min-Table/0},@c421a74a80] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:3 StoreID:3 ReplicaID:2} {NodeID:2 StoreID:2 ReplicaID:3}]
I170209 08:16:46.684866 14340 storage/replica_raftstorage.go:414  [replicate,n1,s1,r6/1:/{Table/14-Max},@c420839500] generated preemptive snapshot c036d85d at index 14
I170209 08:16:46.686135 14340 storage/store.go:3278  [replicate,n1,s1,r6/1:/{Table/14-Max},@c420839500] streamed snapshot: kv pairs: 10, log entries: 4, 1ms
I170209 08:16:46.688064 15090 storage/replica_raftstorage.go:596  [n2,s2,r6/?:{-},@c421a74000] applying preemptive snapshot at index 14 (id=c036d85d, encoded size=1694, 1 rocksdb batches, 4 log entries)
I170209 08:16:46.688238 15090 storage/replica_raftstorage.go:604  [n2,s2,r6/?:/{Table/14-Max},@c421a74000] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170209 08:16:46.688717 14340 storage/replica_command.go:3253  [replicate,n1,s1,r6/1:/{Table/14-Max},@c420839500] change replicas (remove {2 2 3}): read existing descriptor range_id:6 start_key:"\226" end_key:"\377\377" replicas:<node_id:1 store_id:1 replica_id:1 > replicas:<node_id:3 store_id:3 replica_id:2 > next_replica_id:3 
I170209 08:16:46.691982 14876 storage/replica.go:2476  [n1,s1,r6/1:/{Table/14-Max},@c420839500] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:3 StoreID:3 ReplicaID:2} {NodeID:2 StoreID:2 ReplicaID:3}]
I170209 08:16:46.695452 14340 storage/replica_raftstorage.go:414  [replicate,n1,s1,r4/1:/Table/1{2-3},@c421af4700] generated preemptive snapshot 8e6b3f22 at index 24
I170209 08:16:46.695957 14340 storage/store.go:3278  [replicate,n1,s1,r4/1:/Table/1{2-3},@c421af4700] streamed snapshot: kv pairs: 31, log entries: 14, 0ms
I170209 08:16:46.696213 15124 storage/replica_raftstorage.go:596  [n3,s3,r4/?:{-},@c421ef7880] applying preemptive snapshot at index 24 (id=8e6b3f22, encoded size=9271, 1 rocksdb batches, 14 log entries)
I170209 08:16:46.696441 15124 storage/replica_raftstorage.go:604  [n3,s3,r4/?:/Table/1{2-3},@c421ef7880] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170209 08:16:46.696953 14340 storage/replica_command.go:3253  [replicate,n1,s1,r4/1:/Table/1{2-3},@c421af4700] change replicas (remove {3 3 3}): read existing descriptor range_id:4 start_key:"\224" end_key:"\225" replicas:<node_id:1 store_id:1 replica_id:1 > replicas:<node_id:2 store_id:2 replica_id:2 > next_replica_id:3 
I170209 08:16:46.702057 15081 storage/replica.go:2476  [n1,s1,r4/1:/Table/1{2-3},@c421af4700] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2} {NodeID:3 StoreID:3 ReplicaID:3}]
I170209 08:16:46.703074 14340 storage/replica_raftstorage.go:414  [replicate,n1,s1,r2/1:/Table/{0-11},@c420838e00] generated preemptive snapshot 46928d4d at index 27
I170209 08:16:46.707596 14340 storage/store.go:3278  [replicate,n1,s1,r2/1:/Table/{0-11},@c420838e00] streamed snapshot: kv pairs: 34, log entries: 17, 4ms
I170209 08:16:46.707861 14988 storage/replica_raftstorage.go:596  [n2,s2,r2/?:{-},@c421ae6700] applying preemptive snapshot at index 27 (id=46928d4d, encoded size=13647, 1 rocksdb batches, 17 log entries)
I170209 08:16:46.708101 14988 storage/replica_raftstorage.go:604  [n2,s2,r2/?:/Table/{0-11},@c421ae6700] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170209 08:16:46.708958 14340 storage/replica_command.go:3253  [replicate,n1,s1,r2/1:/Table/{0-11},@c420838e00] change replicas (remove {2 2 3}): read existing descriptor range_id:2 start_key:"\210" end_key:"\223" replicas:<node_id:1 store_id:1 replica_id:1 > replicas:<node_id:3 store_id:3 replica_id:2 > next_replica_id:3 
I170209 08:16:46.714573 15056 storage/replica.go:2476  [n1,s1,r2/1:/Table/{0-11},@c420838e00] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:3 StoreID:3 ReplicaID:2} {NodeID:2 StoreID:2 ReplicaID:3}]
I170209 08:16:46.718768 14340 storage/replica_raftstorage.go:414  [replicate,n1,s1,r3/1:/Table/1{1-2},@c421a44e00] generated preemptive snapshot b582edcc at index 26
I170209 08:16:46.719275 14340 storage/store.go:3278  [replicate,n1,s1,r3/1:/Table/1{1-2},@c421a44e00] streamed snapshot: kv pairs: 11, log entries: 16, 0ms
I170209 08:16:46.719536 15120 storage/replica_raftstorage.go:596  [n3,s3,r3/?:{-},@c42219a700] applying preemptive snapshot at index 26 (id=b582edcc, encoded size=7530, 1 rocksdb batches, 16 log entries)
I170209 08:16:46.719794 15120 storage/replica_raftstorage.go:604  [n3,s3,r3/?:/Table/1{1-2},@c42219a700] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170209 08:16:46.720161 14340 storage/replica_command.go:3253  [replicate,n1,s1,r3/1:/Table/1{1-2},@c421a44e00] change replicas (remove {3 3 3}): read existing descriptor range_id:3 start_key:"\223" end_key:"\224" replicas:<node_id:1 store_id:1 replica_id:1 > replicas:<node_id:2 store_id:2 replica_id:2 > next_replica_id:3 
I170209 08:16:46.724337 15156 storage/replica.go:2476  [n1,s1,r3/1:/Table/1{1-2},@c421a44e00] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2} {NodeID:3 StoreID:3 ReplicaID:3}]
I170209 08:16:46.895958 15088 sql/event_log.go:95  [client=127.0.0.1:37587,user=root,n1] Event: "create_database", target: 50, info: {DatabaseName:test Statement:CREATE DATABASE test User:root}
I170209 08:16:46.897681 14339 storage/split_queue.go:98  [split,n1,s1,r6/1:/{Table/14-Max},@c420839500] splitting at key /Table/50/0
I170209 08:16:46.897717 14339 storage/replica_command.go:2397  [split,n1,s1,r6/1:/{Table/14-Max},@c420839500] initiating a split of this range at key /Table/50 [r7]
I170209 08:16:47.041622 15152 sql/event_log.go:95  [client=127.0.0.1:44896,user=root,n3] Event: "create_table", target: 51, info: {TableName:test.table_5 Statement:CREATE TABLE IF NOT EXISTS test.table_5 (id INT PRIMARY KEY, val INT) User:root}
I170209 08:16:47.042215 14339 storage/split_queue.go:98  [split,n1,s1,r7/1:/{Table/50-Max},@c423ae1c00] splitting at key /Table/51/0
I170209 08:16:47.042259 14339 storage/replica_command.go:2397  [split,n1,s1,r7/1:/{Table/50-Max},@c423ae1c00] initiating a split of this range at key /Table/51 [r8]
I170209 08:16:47.059773 15192 sql/event_log.go:95  [client=127.0.0.1:37590,user=root,n1] Event: "create_table", target: 52, info: {TableName:test.table_6 Statement:CREATE TABLE IF NOT EXISTS test.table_6 (id INT PRIMARY KEY, val INT) User:root}
I170209 08:16:47.065610 14339 storage/split_queue.go:98  [split,n1,s1,r8/1:/{Table/51-Max},@c42193ea80] splitting at key /Table/52/0
I170209 08:16:47.065654 14339 storage/replica_command.go:2397  [split,n1,s1,r8/1:/{Table/51-Max},@c42193ea80] initiating a split of this range at key /Table/52 [r9]
I170209 08:16:47.081936 14343 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r8/1:/Table/5{1-2},@c42193ea80] generated Raft snapshot 0aa1a1ef at index 15
E170209 08:16:47.082956 14343 storage/queue.go:628  [raftsnapshot,n1,s1,r8/1:/Table/5{1-2},@c42193ea80] [n1,s1,r8/1:/Table/5{1-2}]: change replicas aborted due to failed preemptive snapshot: r8: remote couldn't accept snapshot with error: [n3,s3],r8: cannot apply snapshot: snapshot intersects existing range [n3,s3,r7/2:/{Table/50-Max}]
I170209 08:16:47.280077 14339 storage/split_queue.go:98  [split,n1,s1,r9/1:/{Table/52-Max},@c421e97c00] splitting at key /Table/53/0
I170209 08:16:47.280114 14339 storage/replica_command.go:2397  [split,n1,s1,r9/1:/{Table/52-Max},@c421e97c00] initiating a split of this range at key /Table/53 [r10]
I170209 08:16:47.280948 15151 sql/event_log.go:95  [client=127.0.0.1:44899,user=root,n3] Event: "create_table", target: 53, info: {TableName:test.table_2 Statement:CREATE TABLE IF NOT EXISTS test.table_2 (id INT PRIMARY KEY, val INT) User:root}
I170209 08:16:47.283646 14343 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r9/1:/{Table/52-Max},@c421e97c00] generated Raft snapshot 7fca501f at index 11
E170209 08:16:47.285031 14343 storage/queue.go:628  [raftsnapshot,n1,s1,r9/1:/{Table/52-Max},@c421e97c00] [n1,s1,r9/1:/{Table/52-Max}]: change replicas aborted due to failed preemptive snapshot: r9: remote couldn't accept snapshot with error: [n3,s3],r9: cannot apply snapshot: snapshot intersects existing range; initiated GC: [n3,s3,r8/2:/{Table/51-Max}]
I170209 08:16:47.333284 15235 sql/event_log.go:95  [client=127.0.0.1:43018,user=root,n2] Event: "create_table", target: 54, info: {TableName:test.table_7 Statement:CREATE TABLE IF NOT EXISTS test.table_7 (id INT PRIMARY KEY, val INT) User:root}
I170209 08:16:47.333613 14339 storage/split_queue.go:98  [split,n1,s1,r10/1:/{Table/53-Max},@c420f74e00] splitting at key /Table/54/0
I170209 08:16:47.333649 14339 storage/replica_command.go:2397  [split,n1,s1,r10/1:/{Table/53-Max},@c420f74e00] initiating a split of this range at key /Table/54 [r11]
I170209 08:16:47.336538 14343 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r10/1:/{Table/53-Max},@c420f74e00] generated Raft snapshot f13e88fc at index 11
E170209 08:16:47.337626 14343 storage/queue.go:628  [raftsnapshot,n1,s1,r10/1:/{Table/53-Max},@c420f74e00] [n1,s1,r10/1:/{Table/53-Max}]: change replicas aborted due to failed preemptive snapshot: r10: remote couldn't accept snapshot with error: [n3,s3],r10: cannot apply snapshot: snapshot intersects existing range; initiated GC: [n3,s3,r8/2:/{Table/51-Max}]
I170209 08:16:47.379569 14339 storage/split_queue.go:98  [split,n1,s1,r11/1:/{Table/54-Max},@c422781880] splitting at key /Table/55/0
I170209 08:16:47.379608 14339 storage/replica_command.go:2397  [split,n1,s1,r11/1:/{Table/54-Max},@c422781880] initiating a split of this range at key /Table/55 [r12]
I170209 08:16:47.380670 15234 sql/event_log.go:95  [client=127.0.0.1:43021,user=root,n2] Event: "create_table", target: 55, info: {TableName:test.table_4 Statement:CREATE TABLE IF NOT EXISTS test.table_4 (id INT PRIMARY KEY, val INT) User:root}
E170209 08:16:47.402437 14823 storage/replica_proposal.go:444  [n3,s3,r2/2:/Table/{0-11},@c42094ee00] could not load SystemConfig span: must retry later due to intent on SystemConfigSpan
I170209 08:16:47.403312 16062 storage/raft_transport.go:437  [n3] raft transport stream to node 2 established
I170209 08:16:47.409093 16030 storage/raft_transport.go:437  [n2] raft transport stream to node 3 established
I170209 08:16:47.418189 14343 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r11/1:/{Table/54-Max},@c422781880] generated Raft snapshot 34272ffc at index 11
E170209 08:16:47.418971 14343 storage/queue.go:628  [raftsnapshot,n1,s1,r11/1:/{Table/54-Max},@c422781880] [n1,s1,r11/1:/{Table/54-Max}]: change replicas aborted due to failed preemptive snapshot: r11: remote couldn't accept snapshot with error: [n3,s3],r11: cannot apply snapshot: snapshot intersects existing range; initiated GC: [n3,s3,r8/2:/{Table/51-Max}]
E170209 08:16:47.466800 14822 storage/replica_proposal.go:720  [n3,s3,r2/2:/Table/{0-11},@c42094ee00] could not load SystemConfig span: must retry later due to intent on SystemConfigSpan
I170209 08:16:47.467217 15088 sql/event_log.go:95  [client=127.0.0.1:37587,user=root,n1] Event: "create_table", target: 56, info: {TableName:test.table_9 Statement:CREATE TABLE IF NOT EXISTS test.table_9 (id INT PRIMARY KEY, val INT) User:root}
I170209 08:16:47.485729 14339 storage/split_queue.go:98  [split,n1,s1,r12/1:/{Table/55-Max},@c421dfc000] splitting at key /Table/56/0
I170209 08:16:47.485764 14339 storage/replica_command.go:2397  [split,n1,s1,r12/1:/{Table/55-Max},@c421dfc000] initiating a split of this range at key /Table/56 [r13]
I170209 08:16:47.486036 15216 sql/event_log.go:95  [client=127.0.0.1:43024,user=root,n2] Event: "create_table", target: 57, info: {TableName:test.table_1 Statement:CREATE TABLE IF NOT EXISTS test.table_1 (id INT PRIMARY KEY, val INT) User:root}
I170209 08:16:47.488221 14343 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r12/1:/{Table/55-Max},@c421dfc000] generated Raft snapshot 1f053283 at index 10
E170209 08:16:47.488574 14343 storage/queue.go:628  [raftsnapshot,n1,s1,r12/1:/{Table/55-Max},@c421dfc000] [n1,s1,r12/1:/{Table/55-Max}]: change replicas aborted due to failed preemptive snapshot: r12: remote couldn't accept snapshot with error: [n3,s3],r12: cannot apply snapshot: snapshot intersects existing range; initiated GC: [n3,s3,r8/2:/{Table/51-Max}]
I170209 08:16:47.494491 14339 storage/split_queue.go:98  [split,n1,s1,r13/1:/{Table/56-Max},@c422766700] splitting at key /Table/57/0
I170209 08:16:47.494529 14339 storage/replica_command.go:2397  [split,n1,s1,r13/1:/{Table/56-Max},@c422766700] initiating a split of this range at key /Table/57 [r14]
I170209 08:16:47.497271 14343 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r13/1:/{Table/56-Max},@c422766700] generated Raft snapshot f9291688 at index 11
E170209 08:16:47.497980 14343 storage/queue.go:628  [raftsnapshot,n1,s1,r13/1:/{Table/56-Max},@c422766700] [n1,s1,r13/1:/{Table/56-Max}]: change replicas aborted due to failed preemptive snapshot: r13: remote couldn't accept snapshot with error: [n3,s3],r13: cannot apply snapshot: snapshot intersects existing range; initiated GC: [n3,s3,r8/2:/{Table/51-Max}]
I170209 08:16:47.561670 15193 sql/event_log.go:95  [client=127.0.0.1:37593,user=root,n1] Event: "create_table", target: 58, info: {TableName:test.table_3 Statement:CREATE TABLE IF NOT EXISTS test.table_3 (id INT PRIMARY KEY, val INT) User:root}
I170209 08:16:47.562224 14339 storage/split_queue.go:98  [split,n1,s1,r14/1:/{Table/57-Max},@c4223ca380] splitting at key /Table/58/0
I170209 08:16:47.562261 14339 storage/replica_command.go:2397  [split,n1,s1,r14/1:/{Table/57-Max},@c4223ca380] initiating a split of this range at key /Table/58 [r15]
I170209 08:16:47.564690 14343 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r14/1:/{Table/57-Max},@c4223ca380] generated Raft snapshot 040b7d56 at index 11
E170209 08:16:47.565983 14343 storage/queue.go:628  [raftsnapshot,n1,s1,r14/1:/{Table/57-Max},@c4223ca380] [n1,s1,r14/1:/{Table/57-Max}]: change replicas aborted due to failed preemptive snapshot: r14: remote couldn't accept snapshot with error: [n3,s3],r14: cannot apply snapshot: snapshot intersects existing range; initiated GC: [n3,s3,r8/2:/{Table/51-Max}]
I170209 08:16:47.693628 15217 sql/event_log.go:95  [client=127.0.0.1:44893,user=root,n3] Event: "create_table", target: 59, info: {TableName:test.table_8 Statement:CREATE TABLE IF NOT EXISTS test.table_8 (id INT PRIMARY KEY, val INT) User:root}
I170209 08:16:47.707314 14339 storage/split_queue.go:98  [split,n1,s1,r15/1:/{Table/58-Max},@c421ea7180] splitting at key /Table/59/0
I170209 08:16:47.707349 14339 storage/replica_command.go:2397  [split,n1,s1,r15/1:/{Table/58-Max},@c421ea7180] initiating a split of this range at key /Table/59 [r16]
I170209 08:16:47.715310 14343 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r15/1:/{Table/58-Max},@c421ea7180] generated Raft snapshot 5950096e at index 10
E170209 08:16:47.717366 14343 storage/queue.go:628  [raftsnapshot,n1,s1,r15/1:/{Table/58-Max},@c421ea7180] [n1,s1,r15/1:/{Table/58-Max}]: change replicas aborted due to failed preemptive snapshot: r15: remote couldn't accept snapshot with error: [n3,s3],r15: cannot apply snapshot: snapshot intersects existing range; initiated GC: [n3,s3,r8/2:/{Table/51-Max}]
I170209 08:16:47.723309 16477 util/stop/stopper.go:493  quiescing; tasks left:
1      storage/queue.go:523
1      kv/txn_coord_sender.go:924
W170209 08:16:47.723402 14932 storage/raft_transport.go:443  [n1] raft transport stream to node 3 failed: rpc error: code = 13 desc = transport is closing
W170209 08:16:47.723482 16030 storage/raft_transport.go:443  [n2] raft transport stream to node 3 failed: rpc error: code = 13 desc = transport is closing
W170209 08:16:47.723610 14781 storage/raft_transport.go:443  [n2] raft transport stream to node 1 failed: rpc error: code = 13 desc = transport is closing
W170209 08:16:47.723636 16062 storage/raft_transport.go:443  [n3] raft transport stream to node 2 failed: rpc error: code = 13 desc = transport is closing
W170209 08:16:47.723720 14978 storage/raft_transport.go:443  [n3] raft transport stream to node 1 failed: rpc error: code = 13 desc = transport is closing
I170209 08:16:47.723794 16477 util/stop/stopper.go:493  quiescing; tasks left:
1      storage/queue.go:523
W170209 08:16:47.723863 14778 storage/raft_transport.go:443  [n1] raft transport stream to node 2 failed: rpc error: code = 13 desc = transport is closing
E170209 08:16:47.723954 14339 internal/client/txn.go:341  [split,n1,s1,r15/1:/{Table/58-Max},@c421ea7180] failure aborting transaction: writing transaction timed out or ran on multiple coordinators; abort caused by: node unavailable; try another peer
E170209 08:16:47.724022 14339 storage/queue.go:628  [split,n1,s1,r15/1:/{Table/58-Max},@c421ea7180] unable to split [n1,s1,r15/1:/{Table/58-Max}] at key "/Table/59/0": storage/replica_command.go:2476: split at key /Table/59 failed: node unavailable; try another peer
	create_test.go:191: table 0: could not be created: pq: unexpected value: raw_bytes:"\364U\032\350\003\n\237\001\n\007table_5\0303 2(\0010\000:\004\010\000\020\000B\022\n\002id\020\001\032\006\010\001\020\000\030\000 \0000\000B\023\n\003val\020\002\032\006\010\001\020\000\030\000 \0010\000H\003R!\n\007primary\020\001\030\001\"\002id0\001@\000J\010\010\000\020\000\032\000 \000Z\000`\002j\n\n\010\n\004root\020\002\200\001\001\210\001\003\230\001\000\262\001\032\n\007primary\020\000\032\002id\032\003val \001 \002(\002\270\001\001\302\001\000" timestamp:<wall_time:1486628206925976227 logical:0 > 
	create_test.go:230: expected 10 tables created, only got 9
```

Here's the main error
```
[10:32:17][Step 2/2] --- FAIL: TestParallelCreateTables (5.30s)
[10:32:17][Step 2/2] 	create_test.go:191: table 6: could not be created: pq: unexpected value: raw_bytes:"\322\275^\022\003\n\237\001\n\007table_4\0304 2(\0010\000:\004\010\000\020\000B\022\n\002id\020\001\032\006\010\001\020\000\030\000 \0000\000B\023\n\003val\020\002\032\006\010\001\020\000\030\000 \0010\000H\003R!\n\007primary\020\001\030\001\"\002id0\001@\000J\010\010\000\020\000\032\000 \000Z\000`\002j\n\n\010\n\004root\020\002\200\001\001\210\001\003\230\001\000\262\001\032\n\007primary\020\000\032\002id\032\003val \001 \002(\002\270\001\001\302\001\000" timestamp:<wall_time:1486981717662573090 logical:0 > 
[10:32:17][Step 2/2] 	create_test.go:230: expected 10 tables created, only got 9
[10:32:17][Step 2/2] === RUN   TestParallelCreateConflictingTables
```

--- a/pkg/sql/create_test.go
+++ b/pkg/sql/create_test.go
@@ -250,7 +250,6 @@ func verifyTables(
 // correctly filled.
 func TestParallelCreateTables(t *testing.T) {
 	defer leaktest.AfterTest(t)()
-	t.Skip("#13504 Test is flaky under stress.")
 
 	// This number has to be around 10 or else testrace will take too long to
 	// finish.

